PyRDF/Proxy.py

```diff
@@ -99,14 +102,13 @@ def GetValue(self):
             The value of the current action node, obtained after executing the
             current action node in the computational graph.
         """
-        from PyRDF import current_backend
 
         # Creating a ROOT.TDirectory.TContext in a context manager so that
         # ROOT.gDirectory won't be changed by the event loop execution.
         with _managed_tcontext():
             if not self.proxied_node.value:  # If event-loop not triggered
                 generator = CallableGenerator(self.proxied_node.get_head())
-                current_backend.execute(generator)
+                self.current_backend.execute(generator)
```

Comment:
So now the current backend is an attribute of the proxies? It should be an attribute of the dataframes, right? Or perhaps of the node we are proxying? We need to discuss where the info about the backend is stored.

Reply:
I will reference the attribute of the head node directly

Reply:
ok!